### ISSUE TITLE
http api: count of clients

### ISSUE THREAD

**Issue by thegobot** [ossrs/srs#1362]
What does the clients property in a stream object mean?
```
https://exampla.com/srs_api/api/v1/streams/
{
app: "live"
audio: null
clients: 71
frames: 705343
id: 4861
kbps: {recv_30s: 593, send_30s: 2366}
live_ms: 1557286481138
name: "ludovik"
publish: {active: true, cid: 239190}
recv_bytes: 3790105835
send_bytes: 48693118606
vhost: 4772
}

https://exampla.com/srs_api/api/v1/clients/
for()
if(client.stream == 4861) count++
endfor

count = 5; //Why 5, not 71?

```
Only rtmp compounds are counted or including hls, flv...etc?
I have SRS 2.0.263 with a little changed in https://github.com/ossrs/srs/blob/6e6e996bbaee73473f2ace76b7cb45a065368e7c/trunk/src/app/srs_app_http_api.cpp#L816

```
ret = stat->dumps_clients(data, 0, 5000);
```

**Comment by winlinvip** [comment]
Will support count HLS in future.

**Comment by winlinvip** [comment]
Fixed in SRS 4.0
